Assets/Fungus/Scripts/Commands/SayLocalized.cs

```diff
@@ -0,0 +1,210 @@
+        [SerializeField] protected bool stopVoiceover = true;
+
+        [Tooltip("Wait for the Voice Over to complete before continuing")]
+        [SerializeField] protected bool waitForVO = false;
+
+        //add wait for vo that overrides stopvo
+
+        [Tooltip("Sets the active Say dialog with a reference to a Say Dialog object in the scene. All story text will now display using this Say Dialog.")]
+        [SerializeField] protected SayDialog setSayDialog;
+
+        protected int executionCount;
+
+        #region Public members
+
+        /// <summary>
+        /// Character that is speaking.
+        /// </summary>
+        public virtual Character _Character { get { return character; } }
+
+        /// <summary>
+        /// Portrait that represents speaking character.
+        /// </summary>
+        public virtual Sprite Portrait { get { return portrait; } set { portrait = value; } }
+
+        /// <summary>
+        /// Type this text in the previous dialog box.
+        /// </summary>
+        public virtual bool ExtendPrevious { get { return extendPrevious; } }
+
+        public override void OnEnter()
+        {
+            if (!showAlways && executionCount >= showCount)
+            {
+                Continue();
+                return;
+            }
+
+            executionCount++;
+
+            // Override the active say dialog if needed
+            if (character != null && character.SetSayDialog != null)
+            {
+                SayDialog.ActiveSayDialog = character.SetSayDialog;
+            }
+
+            if (setSayDialog != null)
+            {
+                SayDialog.ActiveSayDialog = setSayDialog;
+            }
+
+            var sayDialog = SayDialog.GetSayDialog();
+            if (sayDialog == null)
+            {
+                Continue();
+                return;
+            }
+
+            var flowchart = GetFlowchart();
+
+            sayDialog.SetActive(true);
+
+            sayDialog.SetCharacter(character);
+            sayDialog.SetCharacterImage(portrait);
+
+            string displayText = storyText;
+
+            var activeCustomTags = CustomTag.activeCustomTags;
+            for (int i = 0; i < activeCustomTags.Count; i++)
+            {
+                var ct = activeCustomTags[i];
+                displayText = displayText.Replace(ct.TagStartSymbol, ct.ReplaceTagStartWith);
+                if (ct.TagEndSymbol != "" && ct.ReplaceTagEndWith != "")
+                {
+                    displayText = displayText.Replace(ct.TagEndSymbol, ct.ReplaceTagEndWith);
+                }
+            }
+
+            string subbedText = flowchart.SubstituteVariables(displayText);
+
+            AudioClip audioTemp = voiceOverClip;
+            // this should come from the localization-object itself
+
+            voiceOverClip = Resources.Load<AudioClip>("Speech/" + GameObject.Find("GameSafe").GetComponent<Flowchart>().GetStringVariable("Language") + "/" + audioTemp.name);
```

Comment:
This depends on the user having a Flowchart with a specific name and a specific named variable in that Flowchart. It would be cleaner to use the existing SetLanguage command and query a Localization object for the active language. See the localization example for reference.